10-May-2022 08:14:32.092 INFO [http-nio-8080-exec-1] org.apache.catalina.core.ApplicationContext.log Ggg:init
10-May-2022 08:14:32.096 INFO [http-nio-8080-exec-1] org.apache.catalina.core.ApplicationContext.log Ggg:doGet
10-May-2022 08:14:32.096 INFO [http-nio-8080-exec-1] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:33:10.257 INFO [Thread-3] org.apache.catalina.core.ApplicationContext.log Ggg:destroy
10-May-2022 09:34:52.519 SEVERE [http-nio-8080-exec-7] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [jsp] в контексте с путем [/Lab4] выбросил исключение
	java.io.IOException: Произошла ошибка при обработке [/afternoon.jsp] в строке [10]

7:     </head>
8:     <body>
9:         <%
10:             BufferedWriter writer = new BufferedWriter(new FileWriter("E:\\4\\Java\\Lab4\\logs\\stdout.txt"));
11:             writer.append("Afternoon");
12:             writer.append("\n");
13:             writer.close();


Stacktrace:
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:494)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
	Caused by: java.io.FileNotFoundException: E:\4\Java\Lab4\logs\stdout.txt (Системе не удается найти указанный путь)
		at java.base/java.io.FileOutputStream.open0(Native Method)
		at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
		at java.base/java.io.FileWriter.<init>(FileWriter.java:66)
		at org.apache.jsp.index_jsp._jspService(index_jsp.java:226)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		... 25 more
10-May-2022 09:35:05.281 SEVERE [http-nio-8080-exec-9] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [jsp] в контексте с путем [/Lab4] выбросил исключение
	java.io.IOException: Произошла ошибка при обработке [/afternoon.jsp] в строке [10]

7:     </head>
8:     <body>
9:         <%
10:             BufferedWriter writer = new BufferedWriter(new FileWriter("E:\\4\\Java\\Lab4\\logs\\stdout.txt"));
11:             writer.append("Afternoon");
12:             writer.append("\n");
13:             writer.close();


Stacktrace:
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:494)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
	Caused by: java.io.FileNotFoundException: E:\4\Java\Lab4\logs\stdout.txt (Системе не удается найти указанный путь)
		at java.base/java.io.FileOutputStream.open0(Native Method)
		at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
		at java.base/java.io.FileWriter.<init>(FileWriter.java:66)
		at org.apache.jsp.index_jsp._jspService(index_jsp.java:226)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		... 25 more
10-May-2022 09:35:07.221 SEVERE [http-nio-8080-exec-10] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [jsp] в контексте с путем [/Lab4] выбросил исключение
	java.io.IOException: Произошла ошибка при обработке [/afternoon.jsp] в строке [10]

7:     </head>
8:     <body>
9:         <%
10:             BufferedWriter writer = new BufferedWriter(new FileWriter("E:\\4\\Java\\Lab4\\logs\\stdout.txt"));
11:             writer.append("Afternoon");
12:             writer.append("\n");
13:             writer.close();


Stacktrace:
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:494)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
	Caused by: java.io.FileNotFoundException: E:\4\Java\Lab4\logs\stdout.txt (Системе не удается найти указанный путь)
		at java.base/java.io.FileOutputStream.open0(Native Method)
		at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
		at java.base/java.io.FileWriter.<init>(FileWriter.java:66)
		at org.apache.jsp.index_jsp._jspService(index_jsp.java:226)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		... 25 more
10-May-2022 09:37:29.499 SEVERE [http-nio-8080-exec-1] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [jsp] в контексте с путем [/Lab4] выбросил исключение
	java.io.IOException: Произошла ошибка при обработке [/afternoon.jsp] в строке [10]

7:     </head>
8:     <body>
9:         <%
10:             BufferedWriter writer = new BufferedWriter(new FileWriter("E:\\4\\Java\\Lab4\\logs\\stdout.txt"));
11:             writer.append("Afternoon");
12:             writer.append("\n");
13:             writer.close();


Stacktrace:
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:494)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
	Caused by: java.io.FileNotFoundException: E:\4\Java\Lab4\logs\stdout.txt (Системе не удается найти указанный путь)
		at java.base/java.io.FileOutputStream.open0(Native Method)
		at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
		at java.base/java.io.FileWriter.<init>(FileWriter.java:66)
		at org.apache.jsp.index_jsp._jspService(index_jsp.java:226)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		... 25 more
10-May-2022 09:46:27.280 INFO [http-nio-8080-exec-6] org.apache.catalina.core.ApplicationContext.log Sss:init
10-May-2022 09:46:27.282 INFO [http-nio-8080-exec-6] org.apache.catalina.core.ApplicationContext.log Sss:doGet
10-May-2022 09:46:27.282 INFO [http-nio-8080-exec-6] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:46:27.295 INFO [http-nio-8080-exec-7] org.apache.catalina.core.ApplicationContext.log Ggg:init
10-May-2022 09:46:27.296 INFO [http-nio-8080-exec-7] org.apache.catalina.core.ApplicationContext.log Ggg:doGet
10-May-2022 09:46:27.297 INFO [http-nio-8080-exec-7] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:46:27.298 SEVERE [http-nio-8080-exec-7] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [Ggg] в контексте с путем [/InP_Lab3] выбросил исключение
	java.lang.NullPointerException: Cannot invoke "String.equalsIgnoreCase(String)" because "_click" is null
		at Ggg.service(Ggg.java:46)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
10-May-2022 09:46:37.730 INFO [http-nio-8080-exec-8] org.apache.catalina.core.ApplicationContext.log Sss:doGet
10-May-2022 09:46:37.730 INFO [http-nio-8080-exec-8] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:46:37.743 INFO [http-nio-8080-exec-9] org.apache.catalina.core.ApplicationContext.log Ggg:doGet
10-May-2022 09:46:37.743 INFO [http-nio-8080-exec-9] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:46:37.744 SEVERE [http-nio-8080-exec-9] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [Ggg] в контексте с путем [/InP_Lab3] выбросил исключение
	java.lang.NullPointerException: Cannot invoke "String.equalsIgnoreCase(String)" because "_click" is null
		at Ggg.service(Ggg.java:46)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
10-May-2022 09:46:45.582 INFO [http-nio-8080-exec-10] org.apache.catalina.core.ApplicationContext.log Sss:doPost
10-May-2022 09:46:45.583 INFO [http-nio-8080-exec-10] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:46:57.396 INFO [http-nio-8080-exec-1] org.apache.catalina.core.ApplicationContext.log Sss:doGet
10-May-2022 09:46:57.397 INFO [http-nio-8080-exec-1] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:46:57.409 INFO [http-nio-8080-exec-3] org.apache.catalina.core.ApplicationContext.log Ggg:doGet
10-May-2022 09:46:57.409 INFO [http-nio-8080-exec-3] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:46:57.410 SEVERE [http-nio-8080-exec-3] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [Ggg] в контексте с путем [/InP_Lab3] выбросил исключение
	java.lang.NullPointerException: Cannot invoke "String.equalsIgnoreCase(String)" because "_click" is null
		at Ggg.service(Ggg.java:46)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
10-May-2022 09:46:59.991 INFO [http-nio-8080-exec-4] org.apache.catalina.core.ApplicationContext.log Sss:doPost
10-May-2022 09:46:59.991 INFO [http-nio-8080-exec-4] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:47:28.094 INFO [http-nio-8080-exec-6] org.apache.catalina.core.ApplicationContext.log Sss:doPost
10-May-2022 09:47:28.094 INFO [http-nio-8080-exec-6] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:47:28.105 INFO [http-nio-8080-exec-7] org.apache.catalina.core.ApplicationContext.log Ggg:doGet
10-May-2022 09:47:28.105 INFO [http-nio-8080-exec-7] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:47:28.106 SEVERE [http-nio-8080-exec-7] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [Ggg] в контексте с путем [/InP_Lab3] выбросил исключение
	java.lang.NullPointerException: Cannot invoke "String.equalsIgnoreCase(String)" because "_click" is null
		at Ggg.service(Ggg.java:46)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
10-May-2022 09:47:56.979 INFO [http-nio-8080-exec-9] org.apache.catalina.core.ApplicationContext.log Sss:doPost
10-May-2022 09:47:56.980 INFO [http-nio-8080-exec-9] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:47:56.990 INFO [http-nio-8080-exec-9] org.apache.catalina.core.ApplicationContext.log Ggg:doPost
10-May-2022 09:47:56.990 INFO [http-nio-8080-exec-9] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:48:47.915 INFO [http-nio-8080-exec-3] org.apache.catalina.core.ApplicationContext.log Sss:doGet
10-May-2022 09:48:47.915 INFO [http-nio-8080-exec-3] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:48:47.924 INFO [http-nio-8080-exec-4] org.apache.catalina.core.ApplicationContext.log Ggg:doGet
10-May-2022 09:48:47.924 INFO [http-nio-8080-exec-4] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:48:47.925 SEVERE [http-nio-8080-exec-4] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [Ggg] в контексте с путем [/InP_Lab3] выбросил исключение
	java.lang.NullPointerException: Cannot invoke "String.equalsIgnoreCase(String)" because "_click" is null
		at Ggg.service(Ggg.java:46)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
10-May-2022 09:49:10.866 INFO [http-nio-8080-exec-5] org.apache.catalina.core.ApplicationContext.log Sss:doGet
10-May-2022 09:49:10.867 INFO [http-nio-8080-exec-5] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:49:10.868 INFO [http-nio-8080-exec-5] org.apache.catalina.core.ApplicationContext.log Ggg:doGet
10-May-2022 09:49:10.868 INFO [http-nio-8080-exec-5] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:49:20.465 INFO [http-nio-8080-exec-6] org.apache.catalina.core.ApplicationContext.log Sss:doGet
10-May-2022 09:49:20.465 INFO [http-nio-8080-exec-6] org.apache.catalina.core.ApplicationContext.log Sss:service
10-May-2022 09:49:20.743 INFO [http-nio-8080-exec-7] org.apache.catalina.core.ApplicationContext.log Ggg:doGet
10-May-2022 09:49:20.743 INFO [http-nio-8080-exec-7] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:49:56.695 INFO [http-nio-8080-exec-1] org.apache.catalina.core.ApplicationContext.log Ggg:doGet
10-May-2022 09:49:56.695 INFO [http-nio-8080-exec-1] org.apache.catalina.core.ApplicationContext.log Ggg:service
10-May-2022 09:57:16.747 SEVERE [http-nio-8080-exec-4] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [jsp] в контексте с путем [/Lab4] выбросил исключение
	java.io.IOException: Произошла ошибка при обработке [afternoon.jsp] в строке [10]

7:     </head>
8:     <body>
9:         <%
10:             BufferedWriter writer = new BufferedWriter(new FileWriter("E:\\4\\Java\\Lab4\\logs\\stdout.txt"));
11:             writer.append("Afternoon");
12:             writer.append("\n");
13:             writer.close();


Stacktrace:
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:494)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
	Caused by: java.io.FileNotFoundException: E:\4\Java\Lab4\logs\stdout.txt (Системе не удается найти указанный путь)
		at java.base/java.io.FileOutputStream.open0(Native Method)
		at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
		at java.base/java.io.FileWriter.<init>(FileWriter.java:66)
		at org.apache.jsp.index_jsp._jspService(index_jsp.java:226)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		... 25 more
10-May-2022 09:57:45.402 SEVERE [http-nio-8080-exec-5] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [jsp] в контексте с путем [/Lab4] выбросил исключение
	java.io.IOException: Произошла ошибка при обработке [afternoon.jsp] в строке [10]

7:     </head>
8:     <body>
9:         <%
10:             BufferedWriter writer = new BufferedWriter(new FileWriter("E:\\4\\Java\\Lab4\\logs\\stdout.txt"));
11:             writer.append("Afternoon");
12:             writer.append("\n");
13:             writer.close();


Stacktrace:
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:494)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
	Caused by: java.io.FileNotFoundException: E:\4\Java\Lab4\logs\stdout.txt (Системе не удается найти указанный путь)
		at java.base/java.io.FileOutputStream.open0(Native Method)
		at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
		at java.base/java.io.FileWriter.<init>(FileWriter.java:66)
		at org.apache.jsp.index_jsp._jspService(index_jsp.java:226)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		... 25 more
10-May-2022 09:57:46.674 SEVERE [http-nio-8080-exec-6] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [jsp] в контексте с путем [/Lab4] выбросил исключение
	java.io.IOException: Произошла ошибка при обработке [afternoon.jsp] в строке [10]

7:     </head>
8:     <body>
9:         <%
10:             BufferedWriter writer = new BufferedWriter(new FileWriter("E:\\4\\Java\\Lab4\\logs\\stdout.txt"));
11:             writer.append("Afternoon");
12:             writer.append("\n");
13:             writer.close();


Stacktrace:
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:494)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
	Caused by: java.io.FileNotFoundException: E:\4\Java\Lab4\logs\stdout.txt (Системе не удается найти указанный путь)
		at java.base/java.io.FileOutputStream.open0(Native Method)
		at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
		at java.base/java.io.FileWriter.<init>(FileWriter.java:66)
		at org.apache.jsp.index_jsp._jspService(index_jsp.java:226)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		... 25 more
10-May-2022 09:57:58.471 SEVERE [http-nio-8080-exec-8] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() для сервлета [jsp] в контексте с путем [/Lab4] выбросил исключение
	java.io.IOException: Произошла ошибка при обработке [afternoon.jsp] в строке [10]

7:     </head>
8:     <body>
9:         <%
10:             BufferedWriter writer = new BufferedWriter(new FileWriter("E:\\4\\Java\\Lab4\\logs\\stdout.txt"));
11:             writer.append("Afternoon");
12:             writer.append("\n");
13:             writer.close();


Stacktrace:
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:494)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
	Caused by: java.io.FileNotFoundException: E:\4\Java\Lab4\logs\stdout.txt (Системе не удается найти указанный путь)
		at java.base/java.io.FileOutputStream.open0(Native Method)
		at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
		at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
		at java.base/java.io.FileWriter.<init>(FileWriter.java:66)
		at org.apache.jsp.index_jsp._jspService(index_jsp.java:226)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		... 25 more
10-May-2022 21:57:48.596 INFO [Thread-3] org.apache.catalina.core.ApplicationContext.log Sss:destroy
10-May-2022 21:57:48.596 INFO [Thread-3] org.apache.catalina.core.ApplicationContext.log Ggg:destroy
